package.json

```diff
@@ -140,7 +142,8 @@
     "test": "jest",
     "start": "gulp dev",
     "hot": "gulp hot-dev",
-    "build": "gulp build"
+    "build": "gulp build",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'vendor', 'junction') } catch (e) { }\""
```

Comment:
Is this necessary? Looks like this is doing symlinks to vendor, but where are those symlinks used?

Reply:
This line was also added by the bower-away package to provide symlinks for bower_components as the vendor directory. So that files which are using vendor directory can now use this symlink to follow to the bower components. In our case we don't need so I will remove this command from the package.json